CaseStudiesjQueryJavaScript.md

```diff
@@ -176,3 +178,120 @@ $(document).ready(function() {
 </tr>
 
 </table>
+<br>
+
+## 5. Should avoid using duplicate class/ID html attribute
+
+<table>
+<tr id="5.1">
+<td width="5%" >
+
+**5.1**
+
+</td>
+<td width="50%">
+The new HTML class/ID is not used in another HTML tag.
+</td>
+
+<td width="45%">
+
+```html
+// File .html
+
+// Bad
+<script>
+// Existing JS event
+$('.btn-change-time').on('click', function() {
+  openCalendarPopup();
+})
+</script>
+
+<!-- Existing button -->
+<button class="btn-change-time" width="50px">
+  Change delivery time
+</button>
+
+<!-- New button -->
+<button class="btn-change-time" width="30px">
+  Cancel order
+</button>
+
+// Good 👍
+<script>
+// Existing JS event
+$('.btn-change-time').on('click', function() {
+  openCalendarPopup();
+})
+</script>
+
+<!-- Existing button -->
+<button class="btn-change-time" width="50px">
+  Change delivery time
+</button>
+
+<!-- New button -->
+<button class="btn-cancel-order" width="30px">
+  Cancel order
+</button>
+```
+
+</td>
+</tr>
+</table>
+
+<table>
+<tr id="5.2">
+<td width="5%" >
+
+**5.2**
+
+</td>
+<td width="50%">
+The new HTML class/ID is used in another HTML tag but does not affect any of the existing logic.
```

Comment:
Sửa lại mô tả như sau:
Use unique IDs for event bindings to ensure distinct functionality and avoid conflicts.